src/server/session.ts

```diff
@@ -89,6 +89,7 @@ module ts.server {
         export var Geterr = "geterr";
         export var NavBar = "navbar";
         export var Navto = "navto";
+        export var Occurrences = "occurrences";

```

Comment:
Yup, just sent that part out. I'm not that familiar with the server so I'm still getting a feel for things.